manim/mobject/types/vectorized_mobject.py

```diff
@@ -611,10 +618,10 @@ def consider_points_equals_2d(self, p0, p1):
         return True
 
     # Information about line
-    def get_cubic_bezier_tuples_from_points(self, points):
+    def get_cubic_bezier_tuples_from_points(self, points: List[Vector]) -> (List[Vector], List[Vector], List[Vector], List[Vector]):
         return np.array(list(self.gen_cubic_bezier_tuples_from_points(points)))
 
-    def gen_cubic_bezier_tuples_from_points(self, points):
+    def gen_cubic_bezier_tuples_from_points(self, points: List[Vector]) -> (List[Vector], List[Vector], List[Vector], List[Vector]):
```

Comment:
We require indexing so Sequence (list, tuple, ...)
```suggestion
    def gen_cubic_bezier_tuples_from_points(self, points: typing.Sequence[Vector]) -> (List[Vector], List[Vector], List[Vector], List[Vector]):
```